proxyd/backend.go

```diff
@@ -780,15 +780,15 @@ func (bg *BackendGroup) Forward(ctx context.Context, rpcReqs []*RPCReq, isBatch
 		rpcReqs, overriddenResponses = bg.OverwriteConsensusResponses(rpcReqs, overriddenResponses, rewrittenReqs)
 	}
 
+	rpcRequestsTotal.Inc()
```

Comment:
for extra safety, should we defer this call?

```suggestion
	defer rpcRequestsTotal.Inc()
```

Reply:
I don't have a major preference. I guess the only concern would be deferring the metric call could make metrics lag a bit. Maybe it could take an extra 5-10s for the metric, all the requests timeout in 5s